src/Elliptic/Systems/Elasticity/Tags.hpp

```diff
@@ -66,5 +67,29 @@ struct Stress : db::SimpleTag {
   using type = tnsr::II<DataVector, Dim>;
 };
 
-}  // namespace Tags
-}  // namespace Elasticity
+/*!
+ * \brief The energy density \f$U\f$ stored in the deformation of the
+ * elastic material.
+ */
+template <size_t Dim>
+struct PotentialEnergy : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+template <typename FluxesComputerType, typename SolutionType>
+struct FluxesComputer : elliptic::Tags::FluxesComputer<FluxesComputerType> {
```

Comment:
this class shouldn't be in this PR (you won't actually need it anymore with #2100)